core/src/test/scala/integration/kafka/api/LogDirFailureTest.scala

```diff
@@ -33,7 +31,7 @@ import org.junit.Assert.assertTrue
 /**
   * Test whether clients can producer and consume when there is log directory failure
   */
-class LogDirFailureTest extends IntegrationTestHarness {
+class LogDirFailureTest() extends IntegrationTestHarness {
```

Comment:
What's the reason for this change? `()` in a class without parameters is generally discouraged.